### ISSUE TITLE
TypeError related to <class 'oxrdflib.OxigraphStore'>.bind

### ISSUE THREAD

**Issue by milvala** [oxigraph/oxrdflib#13]
Hi

When I'm trying to run the code mentioned in [issue 220 in the Oxigraph repository](https://github.com/oxigraph/oxigraph/issues/220) I run into the error pasted below. The code runs, however, and the output seems to be as expected so I don't know if this is really an error or more of a warning?

Could this be related to the latest release of rdflib?

Thanks in advance!

TypeError
```
caught a TypeError, retrying call to <class 'oxrdflib.OxigraphStore'>.bind without override, see https://github.com/RDFLib/rdflib/issues/1880 for more info
Traceback (most recent call last):
  File "/Users/milan/.pyenv/versions/prefect/lib/python3.9/site-packages/rdflib/namespace/__init__.py", line 627, in _store_bind
    return self.store.bind(prefix, namespace, override=override)
TypeError: bind() got an unexpected keyword argument 'override'
caught a TypeError, retrying call to <class 'oxrdflib.OxigraphStore'>.bind without override, see https://github.com/RDFLib/rdflib/issues/1880 for more info
Traceback (most recent call last):
  File "/Users/milan/.pyenv/versions/prefect/lib/python3.9/site-packages/rdflib/namespace/__init__.py", line 627, in _store_bind
    return self.store.bind(prefix, namespace, override=override)
TypeError: bind() got an unexpected keyword argument 'override'
caught a TypeError, retrying call to <class 'oxrdflib.OxigraphStore'>.bind without override, see https://github.com/RDFLib/rdflib/issues/1880 for more info
Traceback (most recent call last):
  File "/Users/milan/.pyenv/versions/prefect/lib/python3.9/site-packages/rdflib/namespace/__init__.py", line 627, in _store_bind
    return self.store.bind(prefix, namespace, override=override)
TypeError: bind() got an unexpected keyword argument 'override'
caught a TypeError, retrying call to <class 'oxrdflib.OxigraphStore'>.bind without override, see https://github.com/RDFLib/rdflib/issues/1880 for more info
Traceback (most recent call last):
  File "/Users/milan/.pyenv/versions/prefect/lib/python3.9/site-packages/rdflib/namespace/__init__.py", line 627, in _store_bind
    return self.store.bind(prefix, namespace, override=override)
TypeError: bind() got an unexpected keyword argument 'override'
caught a TypeError, retrying call to <class 'oxrdflib.OxigraphStore'>.bind without override, see https://github.com/RDFLib/rdflib/issues/1880 for more info
Traceback (most recent call last):
  File "/Users/milan/.pyenv/versions/prefect/lib/python3.9/site-packages/rdflib/namespace/__init__.py", line 627, in _store_bind
    return self.store.bind(prefix, namespace, override=override)
TypeError: bind() got an unexpected keyword argument 'override'
```

**Comment by Tpt** [comment]
Thank you! indeed rdflib 6.2 released a few days ago added an optional parameter to the `bind` abstract method, breaking backward compatbility.

**Comment by Tpt** [comment]
@milvala I have just released v0.3.2 to Pypi that should fix this error.
